src/utilities/network/lookupApi.js

```diff
@@ -156,11 +186,32 @@ const getPrescriberRequestUrl = params => {
   return baseUrl + endpoint + queryString;
 };
 
-const processPatientResponse = async response => {
-  const responseJson = await response.json();
-  const { error } = responseJson;
-  if (error) throw new Error(error);
-  const patientData = responseJson.map(
+const processInsuranceResponse = response => response.map(({
+    ID: id,
+    insuranceProviderID: insuranceProviderId,
+    nameID: nameId,
+    policyNumberFamily,
+    policyNumberPerson,
+    discountRate,
+    expiryDate,
+    isActive: isActive,
```

Comment:
unnecessary rename; should just be `isActive`

Reply:
Oops, fixed in #2671.